docker/cygnus-ngsi/Dockerfile

```
@@ -327,9 +327,11 @@ RUN \
     cd ${CYGNUS_HOME}/cygnus-ngsi && \
     ${MVN_HOME}/bin/mvn ${MAVEN_ARGS} clean && \
     rm -rf /root/.m2 && rm -rf ${MVN_HOME} && rm -rf ${FLUME_HOME}/docs && rm -rf ${CYGNUS_HOME}/doc && rm -f /*.tar.gz && \
-    echo "INFO: Java runtime not needs JAVA_HOME... Unsetting..." && \
-    unset JAVA_HOME && \
-    yum erase -y git java-${JAVA_VERSION}-openjdk-devel python2 && \
+    # FIXME : Fix java errors and warnings
```

I'd suggest to use issue #2133 for all fixes needed due to CentOS. In this sense:

```suggestion
        # FIXME #2113: Fix java errors and warnings. Original script was breaking in CentOS 8
```

---

I just created an Issue for this specific problem: #2116
```suggestion
    # FIXME #2116: Fix java errors and warnings. Original script was breaking in CentOS 8
```